projects/sandbox/train/train/data_structures.py

```diff
@@ -118,8 +153,13 @@ def forward(self, X: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
             max_center_offset=self.trigger_offset,
             coincident=True,
         )
+
+        waveforms, indices = self.channel_swapper(waveforms)
         X[mask] += waveforms
 
+        # make targets negative if they have had a channel swapped
+        if indices is not None:
+            mask[np.where(mask)[0]][indices] = 0
```

Comment:
You should just be able to go `mask[mask][indices] = 0`. In either case, I think you maybe mean to use `torch.where`

Reply:
Ah masking a mask, good call